webapp/src/ts/effects/reports.effects.ts

```diff
@@ -99,6 +100,10 @@ export class ReportsEffects {
             this.reportActions.setVerifyingReport(false);
           }
 
+          if (unreadCount) {
```

Comment:
You need to put this in the `markRead` effect, otherwise the count keeps going down even if I select reports that I've already read. 
In the original code, this call was wrapped in an if: https://github.com/medic/cht-core/blob/master/webapp/src/js/actions/reports.js#L143 that checked if the report is already read, similar to what we do in the `markRead` effect.

Reply:
Oh I see, fixing

Reply:
Done and unit test updated